src/XCCDF/xccdf_session.c

```diff
@@ -1906,6 +1906,27 @@ struct xccdf_rule_result_iterator *xccdf_session_get_rule_results(const struct x
 	return xccdf_result_get_rule_results(session->xccdf.result);
 }
 
+static int system_is_in_bootc_mode(void)
+{
+#ifdef OS_WINDOWS
+	return 0;
+#else
+	FILE *output = popen("bootc status --format json 2>/dev/null | jq \".status.booted\" 2>/dev/null", "r");
```

Comment:
Do we really want to rely on `jq` being available on the system?

Reply:
Great question! I don't know. The `jq` tool is a part of base images for both RHEL and CentOS 9 and 10. That means it should work for our use case. But I haven't checked other systems.

Reply:
I will explore using libyaml and bootc YAML output.

Reply:
Using YAML would require refactoring the current yamlfilecontent_probe code so that it could be reused here.

Instead, we can search for the booted key using substring search, which will be easier to implement.